_inc/client/components/masthead/index.jsx

```diff
@@ -14,6 +14,7 @@ import analytics from 'lib/analytics';
  */
 import { getSiteConnectionStatus } from 'state/connection';
 import { getCurrentVersion, userCanEditPosts } from 'state/initial-state';
+import { imagePath } from 'constants/urls';
```

Comment:
This is not defined when we're building the static.html file for the loading screen, so the console is always giving me a 404 for this path, even though the logo is showing after it loads.